byway/src/main/webapp/destinations.js

```diff
@@ -134,16 +134,16 @@ function addMarker(searchBox){
 function updateLocations(locationData){
     const container = document.getElementById('destinations-container');
     container.innerText = "";
-    let destinationArray= locationData.destinations;
+    let destinationArray = locationData.destinations;
     destinationArray.forEach((destination) => {
       const request = {
-        query: destination,
+        placeId: destination,
         fields: ["name", "photos", "formatted_address", "rating", "business_status"]
```

Comment:
Seems like some of these fields are unused.

Reply:
took them out!